packages/eslint-plugin/rules/camelcase-acronyms.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
 		const reported = [];
 		const acronyms = [
 			'AMP',
+			'GTM',
```

Comment:
@felixarntz I added this to the rule as it seemed like an acronym that should be there.